### DISCUSSION TITLE
How can I achieve the real pen writing effect

### DISCUSSION THREAD

**Discussion by ruei0325** [fabricjs/fabric.js#6992] [Q&A]
I want to make the handwriting effect more realistic
Just like this
![image](https://user-images.githubusercontent.com/26426560/113687855-557ac900-96fb-11eb-9cb1-748471fb48fd.png)

Any ideas or suggestions？

**Comment by asturur** [comment]
How are you getting that effect? with what library? you can check if you can integrate that library with fabricjs.
